### PR TITLE
fix(pipeline): sleep 30 second to delay next reconcile that caused by cluster offline exception

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
sleep 30 second to delay next reconcile that caused by cluster offline exception

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  Fix the bug that sleep 30 second to delay next reconcile that caused by cluster offline exception    |
| 🇨🇳 中文    |  修复了由集群下线导致的流水线异常重试           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/pipeline/providers/reconciler/legacy/reconciler/reconcile.go [hunk 1]
```diff
@@ -76,6 +81,7 @@ func (r *Reconciler) internalReconcileOnePipeline(ctx context.Context, pipelineI
 					err = errors.Errorf("%v", r)
 				}
 				if err != nil {
+					time.Sleep(autoRetryInterval)
```
Comment (sfwn) [inline modules/pipeline/providers/reconciler/legacy/reconciler/reconcile.go:84]: wait after log is better.

Comment (sfwn) [reply modules/pipeline/providers/reconciler/legacy/reconciler/reconcile.go:84]: add todo, it's a temporary fix

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by sfwn** [comment]
wrong title.
Do you retry all exceptions for 30 seconds?

File: modules/pipeline/providers/reconciler/legacy/reconciler/reconcile.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/providers/reconciler/legacy/reconciler/reconcile.go:84]: adjusted and add todo

**Comment by chengjoey** [comment]
> wrong title. Do you retry all exceptions for 30 seconds?

adjusted

**Review by sfwn — APPROVED** [review]

**Pull request merged**